### ISSUE TITLE
Lint task not working on windows

### ISSUE THREAD

**Issue by alisd23** [reactstrap/reactstrap#145]
By the looks of it the `eslint 'src/**'` was not matching any src files on windows. Possibly only tested on MAC?
Changing it to `eslint src/**` uncovers many linting errors.

After removing the quotes on windows I get these 40 eslint errors. No idea why these aren't being picked up on the CI build. I also get about 4000 line break errors but I've filtered those out from the output below for now. We could either ignore the linebreak eslint rule, because git can work any inconsistencies out, or add a fix task which changes them all to LF automatically, to help windows users.

Also, any ideas why the CI task is not picking up these 40 errors?

```
The react/require-extension rule is deprecated. Please use the import/extensions rule from eslint-plugin-import instead.

C:\Dev\reactstrap\src\Button.js
  13:13  error  'children' PropType is defined but prop is never used  react/no-unused-prop-types

C:\Dev\reactstrap\src\ButtonGroup.js
  5:17  error  'aria-label' PropType is defined but prop is never used  react/no-unused-prop-types
  7:9   error  'role' PropType is defined but prop is never used        react/no-unused-prop-types

C:\Dev\reactstrap\src\ButtonToolbar.js
  5:17  error  'aria-label' PropType is defined but prop is never used  react/no-unused-prop-types
  7:9   error  'role' PropType is defined but prop is never used        react/no-unused-prop-types

C:\Dev\reactstrap\src\Col.js
  19:7  error  'xs' PropType is defined but prop is never used  react/no-unused-prop-types
  20:7  error  'sm' PropType is defined but prop is never used  react/no-unused-prop-types
  21:7  error  'md' PropType is defined but prop is never used  react/no-unused-prop-types
  22:7  error  'lg' PropType is defined but prop is never used  react/no-unused-prop-types
  23:7  error  'xl' PropType is defined but prop is never used  react/no-unused-prop-types

C:\Dev\reactstrap\src\Dropdown.js
  14:10  error  'group' PropType is defined but prop is never used      react/no-unused-prop-types
  16:9   error  'size' PropType is defined but prop is never used       react/no-unused-prop-types
  17:8   error  'tag' PropType is defined but prop is never used        react/no-unused-prop-types
  21:14  error  'className' PropType is defined but prop is never used  react/no-unused-prop-types

C:\Dev\reactstrap\src\DropdownItem.js
  5:13  error  'children' PropType is defined but prop is never used  react/no-unused-prop-types

C:\Dev\reactstrap\src\DropdownMenu.js
  5:13  error  'children' PropType is defined but prop is never used  react/no-unused-prop-types

C:\Dev\reactstrap\src\DropdownToggle.js
   7:13  error  'children' PropType is defined but prop is never used       react/no-unused-prop-types
  11:18  error  'data-toggle' PropType is defined but prop is never used    react/no-unused-prop-types
  12:20  error  'aria-haspopup' PropType is defined but prop is never used  react/no-unused-prop-types

C:\Dev\reactstrap\src\Form.js
  5:13  error  'children' PropType is defined but prop is never used  react/no-unused-prop-types

C:\Dev\reactstrap\src\FormFeedback.js
  5:13  error  'children' PropType is defined but prop is never used  react/no-unused-prop-types

C:\Dev\reactstrap\src\FormGroup.js
  5:13  error  'children' PropType is defined but prop is never used  react/no-unused-prop-types

C:\Dev\reactstrap\src\FormText.js
  5:13  error  'children' PropType is defined but prop is never used  react/no-unused-prop-types

C:\Dev\reactstrap\src\Input.js
  5:13  error  'children' PropType is defined but prop is never used  react/no-unused-prop-types

C:\Dev\reactstrap\src\Label.js
  20:13  error  'children' PropType is defined but prop is never used  react/no-unused-prop-types
  28:7   error  'xs' PropType is defined but prop is never used        react/no-unused-prop-types
  29:7   error  'sm' PropType is defined but prop is never used        react/no-unused-prop-types
  30:7   error  'md' PropType is defined but prop is never used        react/no-unused-prop-types
  31:7   error  'lg' PropType is defined but prop is never used        react/no-unused-prop-types
  32:7   error  'xl' PropType is defined but prop is never used        react/no-unused-prop-types

C:\Dev\reactstrap\src\Media.js
  7:13  error  'children' PropType is defined but prop is never used  react/no-unused-prop-types

C:\Dev\reactstrap\src\Modal.js
  186:9  error  Expected 'this' to be used by class method 'render'  class-methods-use-this

C:\Dev\reactstrap\src\Nav.js
  6:13  error  'disabled' PropType is defined but prop is never used  react/no-unused-prop-types

C:\Dev\reactstrap\src\Navbar.js
  10:9  error  'role' PropType is defined but prop is never used  react/no-unused-prop-types

C:\Dev\reactstrap\src\NavbarToggler.js
  6:9  error  'type' PropType is defined but prop is never used  react/no-unused-prop-types

C:\Dev\reactstrap\src\NavDropdown.js
  6:13  error  'children' PropType is defined but prop is never used  react/no-unused-prop-types

C:\Dev\reactstrap\src\Pagination.js
  5:13  error  'children' PropType is defined but prop is never used  react/no-unused-prop-types

C:\Dev\reactstrap\src\PaginationItem.js
  6:13  error  'children' PropType is defined but prop is never used  react/no-unused-prop-types

C:\Dev\reactstrap\src\Tag.js
  8:13  error  'children' PropType is defined but prop is never used  react/no-unused-prop-types

C:\Dev\reactstrap\src\TetherContent.js
  139:9  error  Expected 'this' to be used by class method 'render'  class-methods-use-this

✖ 40 problems (40 errors, 0 warnings)
```

**Comment by TheSharpieOne** [comment]
You have the wrong version of eslint-config-airbnb. You have [v11.0.0+](https://github.com/airbnb/javascript/blob/master/packages/eslint-config-airbnb/CHANGELOG.md#1100--2016-09-08). Not sure how you got it, the [package.json for this project has `^10.0.0`](https://github.com/reactstrap/reactstrap/blob/master/package.json#L79).
Are you using a global eslint package instead of the project one?

v11 has some breaking changes, including `react/no-unused-prop-types` which you are seeing here.

**Comment by alisd23** [comment]
Ah yes very true, I forgot I tried upgrading to fix a different issue with the atom-linter. :-1: 
So how do you suggest we deal with line endings? Enforce the rule and have some fix command for windows users? Or remove the line ending rule, and let git work it out?

**Comment by eddywashere** [comment]
closed with #146
